### ISSUE TITLE
CRAI: incompatible with BAI metadata

### ISSUE THREAD

**Issue by vadimzalunin** [samtools/hts-specs#146]
Basically CRAI is missing number of mapped/unmapped reads per slice. This prevents accurate conversion from CRAI to BAI.

**Comment by jkbonfield** [comment]
I am not sure the specification ever assumes that you'd want to convert .crai to .bai as it is meant to be a self contained specification.  What is the logic of performing this conversion?  Is this simply an in-memory structure due to how htsjdk works, rather than for creation of an on-disk bai to use instead?

**Comment by vadimzalunin** [comment]
@jkbonfield yes, a common index API to make life easier. I am not so convinced this is required but if CRAI version will ever change then we may as well add these numbers.
